src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/LinkedEditingRangeHandler.cs

```diff
@@ -133,6 +133,9 @@ public async Task<LinkedEditingRanges> HandleRequestAsync(LinkedEditingRangePara
                 return null;
             }
 
+            // Add Razor's Tag Helper opt-out operator to the word pattern
+            linkedEditingRangeResult.WordPattern = linkedEditingRangeResult.WordPattern.Insert(0, "[!?]");
```

Comment:
Hmm, this is a little fragile depending on how the editing range result pattern is constructed. For instance, if WebTools' pattern is: `<[a-z]|<[_a-zA-Z]` then your pattern will only influence the first half.

We can take this approach but if we are to do that we should add some Debugger asserts to validate the structure over the word pattern so we can know if it breaks at dev time.

```suggestion
            linkedEditingRangeResult.WordPattern = linkedEditingRangeResult.WordPattern.Insert(0, "!?");
```